inc/classes/class-assets.php

```diff
@@ -0,0 +1,163 @@
+<?php
+/**
+ * Theme bootstrap file.
+ *
+ * @package Elementary-Theme
+ */
+
+namespace Elementary_Theme;
+
+use Elementary_Theme\Traits\Singleton;
+
+/**
+ * Class Assets
+ *
+ * @since 1.0.0
+ */
+class Assets {
+
+	use Singleton;
+
+	/**
+	 * Constructor.
+	 */
+	protected function __construct() {
+		// Setup hooks.
+		$this->setup_hooks();
+
+	}
+
+	/**
+	 * Setup hooks.
+	 *
+	 * @since 1.0.0
+	 */
+	public function setup_hooks() {
+		add_action( 'wp_enqueue_scripts', [ $this, 'register_assets' ] );
+		add_filter( 'render_block', array( $this, 'enqueue_block_specific_assets' ), 10, 2 );
```

Comment:
I think we should use consistent array syntax and most probably short array syntax. Thoughts?

Reply:
Totally agree!